### ISSUE TITLE
Meta data GPS Latitude and Longitude data returns null

### ISSUE THREAD

**Issue by leapit** [Sh1d0w/multi_image_picker#152]
### Bug Report

As [https://sh1d0w.github.io/multi_image_picker/#/metadata](https://sh1d0w.github.io/multi_image_picker/#/metadata) steps

```
Metadata metadata = await asset.requestMetadata();
print(metadata.gps.gpsLatitude);
print(metadata.gps.gpsLongitude);
print(metadata.exif.artist);
print(metadata.device.model);
```

Console out  `meta.gps.gpsLatitude` and `meta.gps.gpsLongitude}` is null,but strange`meta.gps.gpsLatitudeRef`and `meta.gps.gpsLongitudeRef` are correct.

Just as stackoverflow describe : [https://stackoverflow.com/questions/55815684/why-meta-gps-gpslatitude-is-null](https://stackoverflow.com/questions/55815684/why-meta-gps-gpslatitude-is-null)

Is it happening on Android: Yes 
Is it happening on iOS: Yes

### Flutter Env
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.5.4-hotfix.2, on Mac OS X 10.13.6 17G6030, locale en-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.33.1)
[✓] Connected device (1 available)
```

### multi_image_picker version

`4.3.1`

**Comment by Sh1d0w** [comment]
Are you sure the image you are testing with actually have this metadata?

If you test with some real images that have this metadata it works just fine, try some of those https://github.com/ianare/exif-samples/tree/master/jpg/gps

Please make sure the images you are testing with have this metadata set and also if you are taking them with camera - that it has GPS location enabled.

**Comment by leapit** [comment]
@Sh1d0w  Sure,the image has gps data in fact,and I test the image via online exif tools,got Latitude and Longitude,also I test the image you above (https://github.com/ianare/exif-samples/blob/master/jpg/gps/DSCN0010.jpg) copy original to my phone,got null,but can read `meta.gps.gpsLatitudeRefand`and `meta.gps.gpsLongitudeRef`

**Comment by leapit** [comment]
I test it many times,same result here.I use real phone,not simulator.

**Comment by Sh1d0w** [comment]
This should be fixed in version 4.3.2 by this PR https://github.com/Sh1d0w/multi_image_picker/pull/153

**Comment by leapit** [comment]
> This should be fixed in version 4.3.2 by this PR #153

nice work,fixed  gpsLatitude and gpsLongitude!

but still many gps meta properties returns null,seems strange

```
hashCode:595233798
runtimeType:Type (GpsMetadata)
gpsVersionID:""
gpsLatitudeRef:"N"
gpsLatitude:30.277427
gpsLongitudeRef:"E"
gpsLongitude:120.065478
gpsAltitudeRef:"1.0"
gpsAltitude:null
gpsTimeStamp:"03:07:12"
gpsSatellites:null
gpsStatus:null
gpsMeasureMode:null
gpsDOP:null
gpsSpeedRef:null
gpsSpeed:null
gpsTrackRef:null
gpsTrack:null
```
